docs/io.cozy.accounts.md

```diff
@@ -50,6 +50,23 @@ Generally, the konnector should be able to handle by itself this kind of relatio
 }
 ```
 
+### Vault Cipher
+
+An account can be synced in a password manager. In that case, the `vaultCipher` relationship can be used to store synchronization data.
+
+#### Example
+```json
+{
+  "relationships": {
+    "vaultCipher": {
+      "_id": "123abc",
+      "_doctype": "com.bitwarden.ciphers",
```

Comment:
I think it should be `_type`, and not `_doctype`, as it is explained in the [README](https://github.com/cozy/cozy-doctypes/blob/master/docs/README.md#relationships).